fireant/utils.py

```diff
@@ -13,6 +13,34 @@ def deep_get(d, keys, default=None):
         d_level = d_level[key]
     return d_level
 
+
+def setdeepattr(d, key, value):
```

Comment:
Naming here and in the function below could be improved. Set deep attribute is not really meaningful. Docstrings are welcome as well.

Reply:
will add some docs. The naming is meant to extend `setattr` which is a built in func.